src/opensles/AudioStreamOpenSLES.cpp

```diff
@@ -276,6 +282,7 @@ void AudioStreamOpenSLES::processBufferCallback(SLAndroidSimpleBufferQueueItf bq
 
 // this callback handler is called every time a buffer needs processing
 static void bqCallbackGlue(SLAndroidSimpleBufferQueueItf bq, void *context) {
+    // Any chance we could use a more distinct type here instead of `void *`?
```

Comment:
The void * is specified by the OpenSL ES API. It cannot be changed.
https://www.khronos.org/registry/OpenSL-ES/api/1.1/OpenSLES.h
It is like passing a void * context to pthread_create().